.babelrc

```diff
@@ -1,3 +1,12 @@
 {
- "presets": ["@babel/preset-env"]
+ "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "8.11"
```

Comment:
would we want to do > 8.11 as well?

Reply:
Yeah, that is what this should mean, basically they all are supersets of the earliest one. I tested this on 8.11, 10.19, and 12.13